syntax = "proto3";

package achievements;

service Achievements {
    rpc UpdateAchievements (UpdateAchievementRequest) returns (UpdateAchievementResponse) {}
}

message UpdateAchievementRequest {
    string achievement_id = 1;
    bool completed = 2;
}

message UpdateAchievementResponse {
    string message = 1;
}